### ISSUE TITLE
batchUploadFiles responseBlock called twice when access token is invalid

### ISSUE THREAD

**Issue by rolandpersson** [dropbox/dropbox-sdk-obj-c#77]
As I understand the documentation the responseBlock should only be called once.

But, when attempting to do a batch upload of a single file on iOS and have an invalid access token I receive two calls as below:
```
finish:(null) pollError:(null) requestError:DropboxAuthError[{
    ErrorContent = "invalid_access_token/";
    RequestId = ef382b92ef9044106a8af97a762f5768;
    StatusCode = 401;
    StructuredAuthError = "{\n    \".tag\" = \"invalid_access_token\";\n}";
}];
finish:(null) pollError:(null) requestError:DropboxAuthError[{
    ErrorContent = "invalid_access_token/..";
    RequestId = 3f00d3513bde71d361d6060cfd09153d;
    StatusCode = 401;
    StructuredAuthError = "{\n    \".tag\" = \"invalid_access_token\";\n}";
}];
```

**Comment by greg-db** [comment]
Thanks for the report! We'll look into it.

**Comment by scobbe** [comment]
@rolandpersson: This is fixed now in [v3.0.3](https://github.com/dropbox/dropbox-sdk-obj-c/releases/tag/3.0.3) of the SDK. Apologies for the long delay.
